test/helpers.ts

```diff
@@ -300,6 +300,16 @@ async function performDAOAction(
   );
 }
 
+const validateArraysEqual = async (arrayA: string[], arrayB: string[]) => {
```

Comment:
One-liner:

`a.every((a) => expect(b.map((b) => b.toLowerCase()).includes(a.toLowerCase()))
`

Don't think we should add this function in.